.github/workflows/test_module.yml

```diff
@@ -104,3 +104,60 @@ jobs:
             /home/runner/_work/pytest_workflow_*/**/.command.sh
             /home/runner/_work/pytest_workflow_*/**/.command.run
             !${{ github.workspace }}/.singularity
+
+  nf-test:
+    runs-on: scilus-nf-scil-runners
+    name: nf-test-${{ inputs.module }}
+    if: inputs.module != ''
+    env:
+      NXF_ANSI_LOG: false
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pip pytest-workflow cryptography
+
+      - uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: "17"
+
+      - name: Setup Nextflow
+        uses: nf-core/setup-nextflow@v2
+
+      - name: Setup nf-test
+        uses: nf-core/setup-nf-test@v1
+
+      - name: Setup apptainer
+        if: inputs.profile == 'singularity'
+        uses: eWaterCycle/setup-apptainer@main
+
+      - name: Set up Singularity
+        if: inputs.profile == 'singularity'
+        run: |
+          mkdir -p $NXF_SINGULARITY_CACHEDIR
+          mkdir -p $NXF_SINGULARITY_LIBRARYDIR
+
+      - name: Pull scilus image into cache
+        if: inputs.profile == 'singularity'
+        run: |
+          apptainer pull docker://scilus/scilus:1.6.0
```

Comment:
You can remove (or comment with a message) those lines for now. We skip singularity/apptainer testing for now because our workers cannot handle it. Simply, we don't have a way yet to share a common apptainer cache, and end up saving and building too much copies of the same thing at the same time and burst the disk space.